web/src/components/BackNavigationHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react'
```

Comment:
yeah maybe it would make sense to discuss this again, but lets open an other ticket for it, as this ticket was `Ready` and groomed properly and is now done. So there is not less effort if doing it later and we can avoid having this ticket open for long time and therefore less conflicts to fix.

Reply:
I think we can do a small vote on mattermost